project/__main__.py

```diff
@@ -1 +1,10 @@
-print("exec sources directory")
+from project.console_app import init_console
+
+if __name__ == "__main__":
+    """
+    Main function, console application runs there
+    Returns
```

Comment:
Между смысловыми объектами документации должна быть одна пустая строка
```suggestion
    Main function, console application runs there

    Returns
```

Reply:
И так везде

Reply:
Поправлено